src/content/docs/apis/intro-apis/new-relic-api-keys.mdx

```diff
@@ -292,7 +305,7 @@ Besides our [main API keys explained above](#overview-keys), we have several oth
       To find and manage REST API keys: From the [user menu](/docs/accounts/accounts-billing/general-account-settings/intro-account-settings), click **API keys** (get [a direct link to the API keys page](https://one.newrelic.com/launcher/api-keys-ui.api-keys-launcher)). Then click **REST API key**. Before you configure or delete an API key, ensure you are doing so for the correct account.
   </Collapser>
 </CollapserGroup>
+  </Collapser>
+</CollapserGroup>
 
-## Account ID [#account-id]
```

Comment:
How many clicks from this doc go into the account ID doc? I think it may be a more common path than we think.

Reply:
I'm not sure, but I do have the same link in the information section (we just got rid of the subsection header since it seemed like it didn't belong. Happy to change it back if you like, either now or after we let it ride for a bit and see if the change makes any difference in the analytics.